packages/docs/src/pages/readMe.md

```diff
@@ -136,3 +140,17 @@ curl -X GET 'localhost/meditor/api/me' \
 ```
 
 A user will be returned if your session is still active.
+
+#### Earthdata Login
+
+To log in through the mEditor API, you'll need to send a curl request to mEditor's login endpoint. The example below assumes that you have a `.netrc` file set up with your Earthdata Login (EDL) credentials. General information can be found on `.netrc` can be found in [cURL's documentation](https://everything.curl.dev/usingcurl/netrc); EDL-specific `.netrc` information can be found in the first part Earthdata Login's documentation on [How To Access Data With cURL And Wget](https://urs.earthdata.nasa.gov/documentation/for_users/data_access/curl_and_wget).
+
+```sh
+curl -b ~/.edl_cookies -c ~/.edl_cookies -L -n http://localhost/meditor/api/login > /dev/null
```

Comment:
@benforshey The official Earthdata Login docs call these `.urs_cookies`. I'm not sure what the realistic impact of this would be, but I would prefer us sticking to their convention so we aren't potentially causing issues with other scripts.